src/runtime-rs/crates/hypervisor/src/qemu/inner.rs

```diff
@@ -140,6 +137,9 @@ impl QemuInner {
         }
 
         self.qemu_process = Some(command.stderr(Stdio::piped()).spawn()?);
+        // Close the file descriptors now that QEMU was forked. This must
+        // be done here.
+        fds_for_qemu.clear();
```

Comment:
`fds_for_qemu` is no longer used, I think just dropping it, e.g. `drop(fds_for_qemu)`.

Reply:
This is actually not necessary, the `File`s will be closed implicitly when this function exits.

Reply:
> `fds_for_qemu` is no longer used, I think just dropping it, e.g. `drop(fds_for_qemu)`.

Yeah even better.

> This is actually not necessary, the `File`s will be closed implicitly when this function exits.

Yes I know but the idea is to make this explicit.

Reply:
Why would we want to do so?  The whole idea of destructors is to make this implicit.  You should only `drop()` explicitly when there's a good reason to do so, otherwise it's confusing to the reader.

Reply:
Because I don't really know when the destructor will be eventually called but I do know that we want the fds to be closed as early as possible... ideally, we'd even close them before the child process execs into QEMU.

Reply:
Finally dropped this line after discussion with @pmores on slack.